packages/sit-onyx/src/components/OnyxCheckbox/OnyxCheckbox.stories.ts

```diff
@@ -56,6 +56,15 @@ export const Required = {
     ...Default.args,
     required: true,
   },
+  decorators: [
```

Comment:
Hmm the example looks a little off when the tooltip is not shown
![image](https://github.com/user-attachments/assets/55fdff25-e5c8-4542-a514-76319ab9645b)

Reply:
as discussed, I added a point about that in 
- #730
:)